src/options.js

```diff
@@ -331,7 +331,7 @@ export function isTemporalString(values) {
 export function isNumericString(values) {
   for (const value of values) {
     if (value == null || value === "") continue;
```

Comment:
Given that we already ignore the empty string, I wonder if we shouldn’t ignore all whitespace strings, too?

```suggestion
    if (value == null || (typeof value === "string" && /^\s*$/.test(value))) continue;
```

Then you won’t need to trim below, and it would mean that we continue checking the other values if we encounter a whitespace string (rather than returning false).